.travis.yml

```diff
@@ -1,37 +1,57 @@
----
-sudo: false
 language: rust
+sudo: required
```

Comment:
Agreed.  It is annoying -- I removed some combos from the matrix to make overall build time somewhat reasonable, but that is a tradeoff and we could, for instance, miss a failure on some intermediate rust version.